pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -333,7 +333,14 @@ func GatherClusterOperators(i *Gatherer) func() ([]record.Record, []error) {
 		}
 		records := make([]record.Record, 0, len(config.Items))
 		for index := range config.Items {
-			records = append(records, record.Record{Name: fmt.Sprintf("config/clusteroperator/%s", config.Items[index].Name), Item: ClusterOperatorAnonymizer{&config.Items[index]}})
+			recordName := fmt.Sprintf("config/clusteroperator/%s", config.Items[index].Name)
```

Comment:
clusteroperators are also not namespace specific, but they are related to namespaces, where the pods or events are NS specific.